src/main/java/io/valkey/DefaultJedisSocketFactory.java

```diff
@@ -67,7 +67,7 @@ private Socket connectToFirstSuccessfulHost(HostAndPort hostAndPort) throws Exce
 
         socket.setReuseAddress(true);
         socket.setKeepAlive(true); // Will monitor the TCP connection is valid
-        socket.setTcpNoDelay(true); // Socket buffer Whetherclosed, to ensure timely delivery of data
```

Comment:
This is a very old corruption:

https://github.com/redis/jedis/commit/a688c610ae6d4927e9778833921950b7c04ff01a